### PR TITLE
utils: Add --enforce-signature option for eval, guide, and fix modules

### PR DESCRIPTION
**Pull request by evgenyz**
With this option set all unsigned data streams would be considered invalid. It is the antipode of `--skip-signature-validation` option.

Fixes: #1713.

### CODE REVIEW

File: utils/oscap.8 [hunk 1]
```diff
@@ -191,6 +191,11 @@ Do not validate input/output files.
 Do not validate digital signatures in digitally signed SCAP source data streams.
 .RE
 .TP
+\fB\-\-enforce-signature\fR
+.RS
+Process only digitally signed SCAP source data streams.
```
Comment (jan-cerny) [inline utils/oscap.8:196]: Maybe add for clarity that data streams with no signature will be rejected.

File: src/source/signature.c [hunk 2]
```diff
@@ -159,6 +159,11 @@ static int _oscap_signature_validate_doc(xmlDocPtr doc, oscap_document_type_t sc
 	xmlNodePtr root = xmlDocGetRootElement(doc);
 	node = xmlSecFindNode(root, xmlSecNodeSignature, xmlSecDSigNs);
 	if (node == NULL) {
+		if (enforce_signature) {
+			oscap_seterr(OSCAP_EFAMILY_XML, "Signature not found");
+			dW("Signature node not found");
```
Comment (jan-cerny) [inline src/source/signature.c:164]: It's kind of duplicate.

File: utils/oscap.8 [hunk 3]
```diff
@@ -191,6 +191,11 @@ Do not validate input/output files.
 Do not validate digital signatures in digitally signed SCAP source data streams.
 .RE
 .TP
+\fB\-\-enforce-signature\fR
```
Comment (jan-cerny) [inline utils/oscap.8:194]: Please add it also to `dist/bash_completion.d/oscap`.

**Review by jan-cerny — CHANGES_REQUESTED** [review]
It is great

File: utils/oscap.8 (hunk 3 above)
Comment (evgenyz) [reply utils/oscap.8:194]: Damn, always forgetting it. Thanks.

File: src/source/signature.c (hunk 2 above)
Comment (evgenyz) [reply src/source/signature.c:164]: Well, I agree.

File: utils/oscap.8 (hunk 1 above)
Comment (evgenyz) [reply utils/oscap.8]: Done

**Pull request merged**